src/google_map_markers.js

```diff
@@ -263,10 +263,10 @@ export default class GoogleMapMarkers extends Component {
           child.props.lng === undefined
         ) {
           return React.cloneElement(child, {
-            $geoService: this.props.geoService,
-            $onMouseAllow: this._onMouseAllow,
-            $prerender: this.props.prerender,
```

Comment:
Why are we removing the `$` prefix?

Reply:
Sorry, my misunderstand. I am very interested in this cooperation. Is there any issue let me fix?

Reply:
Sorry again